### PR TITLE
Webpack Optimization

### PR DESCRIPTION
**Pull request by guyyoo**
https://github.com/digi-serve/test_pwa/issues/4

### CODE REVIEW

**Review by nh758 — APPROVED** [review]

File: webpack.common.js [hunk 1]
```diff
@@ -9,9 +9,8 @@ module.exports = {
       app: path.join(APP, "src", "js", "app.js"),
    },
    output: {
-      // path: path.join(APP, "..", "web", "assets", "app"),
       path: path.join(APP, "..", "web", "assets", "mobile"),
-      filename: "mobile_[name].js",
+      filename: "mobile_[name].[contenthash].js",
```
Comment (Hiro-Nakamura) [inline webpack.common.js:13]: ok, so here is the thing.  there is an `api_sails/views/mobile_pwa.js` that needs to reference these `.js` files.  If we put `[contenthash]` on here, then that file needs to be updated each time we compile to reference these new `[contenthash]`es.   

But if we leave it off, we can just continue to use `mobile_runtime`, `mobile_app`, `mobile_vendors`  without worrying about updating that file every time.

**Review by Hiro-Nakamura — CHANGES_REQUESTED** [review]
take a look at the `contenthash` setting

File: webpack.common.js (hunk 1 above)
Comment (guyyoo) [reply webpack.common.js]: https://github.com/digi-serve/test_pwa/pull/8/commits/80639fb046aee8c1dd84e82e7a92c9d03508eb7b

**Review by Hiro-Nakamura — APPROVED** [review]
LGTM

**Pull request merged**